azurerm/internal/services/mssql/mssql_database_resource.go

```diff
@@ -447,7 +447,7 @@ func resourceArmMsSqlDatabaseRead(d *schema.ResourceData, meta interface{}) erro
 		d.Set("auto_pause_delay_in_minutes", props.AutoPauseDelay)
 		d.Set("collation", props.Collation)
 		d.Set("elastic_pool_id", props.ElasticPoolID)
-		d.Set("license_type", props.LicenseType)
+		d.Set("license_type", d.Get("license_type").(string))
```

Comment:
this doesn't track if the values changed outside of Terraform - is this not being returned from the API? if so that's an API bug?

Reply:
Yes. API doesn't return the value of LicenseType. OK. I will close this PR and raise an issue on rest api spec.